.github/workflows/ci.yml

```diff
@@ -19,58 +19,58 @@
 ---
 name: Airflow
```

Comment:
It's probably a nit but maybe we should consider another name like "CI" or "Build"?

Reply:
Airflow CI Build :)

Reply:
Long one :D The name is displayed in the README badge (so in case of change we should update it too)

Reply:
I having both  "Airflow" and "Build" is redundant here, and one or the other would be better

![image](https://user-images.githubusercontent.com/34150/79447778-45447800-7fd8-11ea-8971-179c734abc72.png)

Reply:
CI Build os fine for me :)